Database/homework1.sql

```diff
@@ -0,0 +1,44 @@
+/*Find out how many tasks are in the task table*/
+SELECT COUNT(*) FROM task
+
+/*Find out how many tasks in the task table do not have a valid due date*/
+SELECT * FROM task WHERE due_date IS NULL
+
+/*Find all the tasks that are marked as done*/
+SELECT * FROM task WHERE status_id = 3
+
+/*Find all the tasks that are not marked as done*/
+SELECT * FROM task WHERE status_id != 3
```

Comment:
same here

Reply:
Thank you for your feedback @jpganz, I have now refactored the query